src/ui/MLFacet/MLFacetValues/MLFacetValueRenderer.ts

```diff
@@ -60,9 +62,18 @@ export class MLFacetValueRenderer {
     this.facet.toggleSelectValue(this.facetValue.value);
     this.toggleSelectedClass();
     this.facet.enableFreezeCurrentValuesFlag();
-    this.facet.triggerNewQuery();
+    this.facet.triggerNewQuery(() => this.sendSelectionUAEvent());
```

Comment:
Same comment as above on calling the log function rather than passing it into `triggerNewQuery`.